### PR TITLE
Updated rules to include links to PBIs in todos

### PR DESCRIPTION
**Pull request by matt-goldman-ssw**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

✏️ Conversation with @danielmackay 

> 2. What was changed?

✏️ The rules around `TODOs` were updated - an old rule was archived and another rule was updated to indicate that they should be linked to the backlog.

> 3. Did you do pair or mob programming (list names)?

✏️N/A

### CODE REVIEW

**Review by tiagov8 — APPROVED** [review]

File: rules/create-task-list-comments-for-your-code/rule.md
Comment (danielmackay) [inline rules/create-task-list-comments-for-your-code/rule.md:1]: Do we need to add a redirect from the old page to the new page?

File: rules/todo-tasks/rule.md [hunk 1]
```diff
@@ -8,20 +8,70 @@ authors:
     url: https://www.ssw.com.au/people/adam-cogan
   - title: Matt Wicks
     url: https://www.ssw.com.au/people/matt-wicks
+  - title: Matt Goldman
+    url: https://www.ssw.com.au/people/matt-goldman
 related:
```
Comment (danielmackay) [inline rules/todo-tasks/rule.md:13]: Feeling like adding me to this one? 😁

**Review by danielmackay — APPROVED** [review]
2 comments.

**Review by tiagov8 — APPROVED** [review]

**Comment by tiagov8** [comment]
> 2 comments.

both done

**Pull request merged**